### PR TITLE
a couple of changes to tags.json

### PR DESCRIPTION
**Pull request by Vahvl**
added a [:3] tag to myself

Rayless asked for some things so here they are:
   included tags [GOAT] with the cool colour scheme in the pr for bobert89 and gamingmice
   removed the [cool] tag from billdepzai

message link to what rayless said cuz idk: https://discord.com/channels/780181693100982273/887818760126345246/1153743510701342810

### CODE REVIEW

**Comment by MicrocontrollersDev** [comment]
i would like to veto this

**Comment by nacrt** [comment]
i don't know what veto means but i agree with micro

**Comment by KTibow** [comment]
@nacrt @MicrocontrollersDev i will be merging this pr in 12 hours if you do not leave a "request changes" review

**Comment by Vahvl** [comment]
wait dont do 12h I appareantly did an oopsie with one of the UUIDs

File: docs/assets/tags.json [hunk 1]
```diff
@@ -7,8 +7,10 @@
     "Booster": ["&5[&dBooster&5]", "&5[&dB&5]"],
     "ResourceCreator": ["&1[&9Resource Creator&1]", "&1[&9RC&1]"],
     "cool": ["&9[&5Cool&9]", "&9[&5C&9]"],
+    "GOAT": ["&9[&5GOAT&9]", "&9[&5goat&9]"],
     "nacrt": ["&a[nacrt]", "&a[nacrt]"],
     "micro": ["&b[&dAnime &9Girl &3Enthusiast&b]", "&b[&6Weeb&b]"],
+    "Vahvl": ["&5[:3]&r", "&5[:3]&r"]
```
Comment (MicrocontrollersDev) [inline docs/assets/tags.json:13]: delete this. completely nuke it. level the whole repository.

**Review by MicrocontrollersDev — CHANGES_REQUESTED** [review]

**Comment by Vahvl** [comment]
insert crying spongebob

**Comment by Vahvl** [comment]
also you can do 12h the oopsie i appareantly made is not real

**Comment by MicrocontrollersDev** [comment]
> kti it has been 16h

"if you do not leave a "request changes" review"

**Comment by Vahvl** [comment]
good point, however, i disagree with your opinion

**Comment by Vahvl** [comment]
your [anime girl enthusiast] tag having ass can not be out here complaining abt [:3]

**Comment by MicrocontrollersDev** [comment]
i forgot i had that ngl. regardless it is satirical. meanwhile yours is a warcrime

**Comment by Vahvl** [comment]
is that really true man

**Comment by MicrocontrollersDev** [comment]
yes that was a bit i was really committed too back then. idk how long you’ve known me but i have very clear eras where for a few months i’ll go all in for a bit and i stick with it. take the 2-3 months of keqing for example

**Comment by Zetvue** [comment]
i agree with this. :3 tag

**Comment by Vahvl** [comment]
very based

**Comment by RayDeeUx** [comment]
proposal: merge this pull request and deny future pull requests of this nature

**Comment by MicrocontrollersDev** [comment]
> proposal: merge this pull request and deny future pull requests of this nature

shut up ery

**Comment by KTibow** [comment]
since consensus wasn't reached and this has gone stale i'm closing this. feel free to make another pr though.

**Pull request closed without merging**